test/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 2.8.2)
```

Comment:
Why is this a separate CMake project ? cannot be added to the main one simply by adding the subdirectory?